StackOverflow/Model/QuestionsDataObject.m

```diff
@@ -0,0 +1,34 @@
+//
+//  QuestionsDataObject.m
+//  StackOverflow
+//
+//  Created by Malea on 2017/10/04.
+//  Copyright © 2017 DVT. All rights reserved.
+//
+
+#import "QuestionsDataObject.h"
+#import "DataResult.h"
+
+@implementation QuestionsDataObject
+
+- (void)getData {
```

Comment:
All this class seems to do is forward the messages to the question request. If that's the case why is it needed ?